docs/pages/get-started/faq.md

```diff
@@ -384,109 +411,148 @@ Clients communicate with XMTP nodes through a gRPC (or JSON/HTTP) message API.
 
 XMTP provides perceptibly real-time message delivery and retrieval. The network does not provide service level guarantees.
 
-### Is there a way to get a list of all current XMTP-enabled wallets?
+## Fees
 
-XMTP doesn't provide a direct method to fetch all XMTP-enabled wallet addresses in bulk. You might consider using a third-party service to do so instead of building the functionality yourself.
+XMTP core developers and researchers are working on a specific fee model for XMTP, with the following guiding principles in mind:
 
-For example, services like [Blaze](https://www.withblaze.app/) have compiled extensive lists of XMTP addresses by querying the XMTP `canMessage` API over time at a rate that avoids hitting the limits.
+- Infrastructure costs for the network must remain low even when decentralized, and comparable to the costs for an equivalent centralized messaging service.
+- There must be a sizable free tier for most end users, with the low cost per-user subsidized by the largest apps and bulk senders.
+- There must be a low "take rate": the biggest driver of cost must be infrastructure costs, with any remaining cost returned to the network.
 
-### Does XMTP support group chat?
+Have questions or feedback about the fee model for XMTP? See [XMTP fees: Guiding principles and FAQ](https://community.xmtp.org/t/xmtp-fees-guiding-principles-and-faq/795) in the XMTP Community Forums.
```

Comment:
Looping in @neekolas on this section to get the messaging right, he's much deeper in this than I am